### PR TITLE
Mobile-first: Milestone 3

### PR DESCRIPTION
**Pull request by saluumaa**
In this part I:
1. Added a contact section in the Html file
2. Added a style in the contact section using CSS
3. check the errors using linters

### CODE REVIEW

**Review by amejid — COMMENTED** [review]
Hi team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, that does not have a README file following the provided template. Kindly update your README file using [the template](https://github.com/microverseinc/readme-template) provided by Microverse.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -284,6 +284,81 @@ header a {
...
+  padding: 15px;
+  width: 375px;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  font-size: 17px;
+  font-weight: 400;
+  line-height: 20px;
+  color: #172b4d;
+  outline: none;
+}
```
Comment (ABDULALI3468) [inline style.css]: Your style is not exactly similar to the [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=8P269HoVLs4fzLYn-0). Your width is `375px` where in the template it `is 327px` with the center aligned. Please consider reviewing your styles to make them exact replica of the template so it looks more better and perfect.

File: style.css [hunk 2]
```diff
@@ -284,6 +284,81 @@ header a {
...
+
+.contact-section p {
+  color: #ebebff;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  margin-top: -10px;
+  padding: 30px;
+}
```
Comment (ABDULALI3468) [inline style.css:319]: Please consider reviewing your fonts in accordance with the template fonts. The font family used in the Figma template is `    font-family: 'Poppins';` They might not be available generally so you can download or use the CDN link of Google fonts for them to use.

File: style.css [hunk 3]
```diff
@@ -284,6 +284,81 @@ header a {
...
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  border-radius: 8px;
+  margin-left: 0;
+}
+
 .btn:enabled {
   color: #6070ff;
   background-color: #fff;
```
Comment (ABDULALI3468) [inline style.css]: Please consider reviewing your styles. Try using the following syntax to achieve that the pressed functionality:-

`.btn:active{
  color: #FFFFFF;
  background: #2230D2;
}`

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @saluumaa , 



You did an exceptional job but there are some issues in the styling which need to be addressed to go to the next level.


## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

 ## Required Changes ♻️

**_Check the comments below to see the changes required._**


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by pinkmoon25 — APPROVED** [review]
Hi @saluumaa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
linter checks :heavy_check_mark: 
form implementation :heavy_check_mark: 
HTML validation :heavy_check_mark: 

## Optional suggestions

**- [ ] Please add a license file to your project and fix the MIT link in the readem which currently id giving a 404 error**

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**